demos/python_demos/rag_chatbot/README.md

```diff
@@ -0,0 +1,163 @@
+# LLM-powered RAG chatbot serving via Python Calculator in MediaPipe Graph {#ovms_demo_python_rag_chatbot}
+
+This demo shows how to take advantage of OpenVINO Model Server to generate content remotely with LLM models based off given documents.
+The demo explains how to serve MediaPipe Graph with Python Calculator. In Python Calculator, we use Hugging Face Optimum with OpenVINO Runtime as execution engine. Entire processing is wrapped inside LangChain Retrieval QA pipeline.
+
+Using the gRPC streaming interactive approach allows the user to read the response as they are getting generated.
+
+This demo presents the use case with [tiny-llama-1b-chat](https://huggingface.co/TinyLlama/TinyLlama-1.1B-Chat-v0.1) model but the included python scripts are prepared for several other LLM models. Those are tiny-llama-1b-chat, red-pajama-3b-chat, llama-2-chat-7b, mpt-7b-chat,qwen-7b-chat, chatglm3-6b, mistral-7b, zephyr-7b-beta,neural-chat-7b-v3-1, notus-7b-v1 and youri-7b-chat.
+In this demo the model can be set by:
+```bash
+export SELECTED_MODEL=tiny-llama-1b-chat
+```
+
+## Requirements
+A Linux host with Docker engine installed and sufficient available RAM to load the model and optionally equipped with an Intel GPU card. This demo was tested on a host with Intel® Xeon® Gold 6430 and an Intel® Data Center GPU Flex 170. 
+Running the demo with smaller models like `tiny-llama-1b-chat` requires approximately 4GB of available RAM.
+
+## Build image TODO
```

Comment:
What is missing in this part that makes it `TODO` ?

Reply:
Adjusted